docs/rfcs/0013-rfc-process-updates.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: RFC process updates
+permalink: /docs/rfcs/0013-rfc-process-updates
+---
+
+* RFC PR: /* TODO */
+* Start date: 2024-02-20 
+* End date: 2024-03-22 
+* Stakeholders: @jonalmeida, @boek
+
+## Summary
+
+Clarify requirements for RFCs by introducing a README and a template.
+
+## Motivation
+
+- The scale of the Firefox Android team has grown substantially since the RFC process was first introduced. 
+- More external teams are using and contributing to Android Components. 
+- Approval or rejection of a proposal has required periods of time that can slow implementation.
+- Stakeholders have either been implicit by sharing RFCs in various communication channels, or an author must be expected to correctly select reviewers that lack explicitly documented ownership of affected areas.
+
+## Guide-level explanation
+
+This proposal suggests improving the RFC process by introducing the following:
+
+- an RFC template
+- a guide for when RFCs are appropriate and when they are not needed in the form of a README
+- explicit stakeholders for RFCs
+- an initial deadline for each RFC
+- a new RFC section, "artifacts" which contain links to any additional artifacts introduced alongside the RFC
+- updating our CODEOWNERS file to better facilitate ownership for new Firefox Android "groups" and external teams
+
+Each of these updates are handled as part of this RFC, with the exception of updating the CODEOWNERS file. That will need to undergo a discovery process and communication between Firefox Android groups and other teams.
```

Comment:
I'm not sure if there's a better way to address this issue than reaching out to potentially CODEOWNERS directly but would be interested in a more public approach if anyone has suggestions.